example/lib/main.dart

```diff
@@ -137,6 +140,25 @@ class _ExampleLauncherPageState extends State<ExampleLauncherPage> {
                   child: const Text('Verifications'),
                 ),
                 Text(verificationsResultText),
+                ElevatedButton(
+                  onPressed: _openFamilyOverviewPage,
+                  child: const Text('Family Overview'),
+                ),
+                Text(familyOverviewResultText),
+                ElevatedButton(
+                  onPressed: _openChildDetailPage,
+                  child: const Text('Child Detail Page'),
+                ),
+                Text(childDetailResultText),
+                TextFormField(
+                  onChanged: (text) => childId.add(text),
+                  decoration: const InputDecoration(
+                    border: OutlineInputBorder(),
+                    hintText: 'Child Id (for child detail page)',
+                  ),
+                  keyboardType: TextInputType.text,
+                  autofocus: true,
```

Comment:
Brauchts das autofocus hier? Ist ja eh weiter unten in der Liste, ist vielleicht gar nicht so gut wenn gleich die Tastatur hochkommt obwohl man vielleicht was ganz anderes machen will? Oder bewirkt das was ganz anderes?

Reply:
Ah ich sehe die anderen haben das auch... schadet aber vielleicht nicht nochmal kurz zu schauen ob das überhaupt nötig ist.

Reply:
Ja das ist für die Tastatureingabe im Emulator. Das Feld ist direkt aktiv und wenn Du tippst kommts direkt da rein. Und Du kannst zwischen den Feldern mit Tab wechseln.